test/k8sT/Nightly.go

```diff
@@ -433,12 +445,11 @@ var _ = Describe("NightlyExamples", func() {
 		for _, image := range helpers.NightlyStableUpgradesFrom {
 			func(image string) {
 				It(fmt.Sprintf("Update Cilium from %s to master", image), func() {
-					helpers.InstallExampleCilium(kubectl, image)
+					By("Cleaning up everything before testing an upgrade")
```

Comment:
I would add the image in that message, due there is an image, and not sure what image it is.

Reply:
Done